fsd_utils/locale_selector/get_lang.py

```diff
@@ -4,7 +4,16 @@
 
 
 def get_lang():
-    # get locale from cookie if set
+    
+    # get lang if lang query arg is set
+    language_from_query_args = request.args.get("lang")
+    if language_from_query_args:
+        if language_from_query_args not in ["cy", "en"]:
+            raise ValueError("Invalid language code. Supported codes are 'cy' and 'en'.")
```

Comment:
Can we test how the apps work without this condition locally? If they default to `en` it'd be fine to remove this.

Reply:
Tested as suggested. I guess we need value error for now.